content/go/concepts/conditionals/conditionals.md

```diff
@@ -0,0 +1,103 @@
+---
+Title: 'Conditionals'
+Description: 'Conditionals are used to evaluate if an expression is `true` or `false` and execute code based on that value.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Comparison'
+  - 'Booleans'
+  - 'If'
+  - 'Else'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/back-end-engineer-career-path'
+---
+
+Conditionals are used to evaluate if an expression is `true` or `false` and execute code based on that value. There are several statements that use conditional expressions to determine what code to execute. The primary one being the `if...else if...else` statements.
+
+Other statements that can use conditional statements are [`switch`](https://www.codecademy.com/resources/docs/go/switch) and [`for`](https://www.codecademy.com/resources/docs/go/loops).
+
+## The `if` Statement
+
+One of the statements that's thought of most often in connection with conditional statements is the `if` statement. In its most basic form, the `if` statement evaluates if a condition is `true`, and if so, executes its associated code block.
+
+### Syntax
+
+If `condition` is `true`, execute `statements`. Execution continues after the `if` block regardless of the value of `condition`.
+
+```pseudo
+if condition {
+  statements
+}
+```
+
+## The `else` Statement
+
+The `else` statement provides an optional block of code to execute if the prior `if` (and `else if`) conditions are `false`.
+
+### Syntax
+
+If `condition` is `true`, execute `statements_1`. If `condition` is `false`, execute `statements_2`. Execution then continues after the `else` block.
+
+```pseudo
+if condition {
+  statements_1
+} else {
+  statements_2
+}
+```
+
+## The `else if` Statement
+
+The addition of `else if` statements allow additional conditions to be tested if the prior ones are `false`, each is associated with its own block of code to execute if its condition is `true`.
+
+### Syntax
+
+If `condition_1` is `true`, execute `statements_1`. If `condition_1` is `false` and `condition_2` is `true`, then execute `statements_2`. If neither condition is `true`, execute `statements_3`. Execution then continues after the `else` block. There can be any number of `else if` statements, and the final `else` is optional.
+
+```pseudo
+if condition_1 {
+  statements_1
+} else if condition_2 {
+  statements_2
+} else {
+  statements_3
+}
+```
+
```

Comment:
Let's add a short "Example" section with a regular ```go code snippet. Maybe it could include info about [using `if`s with short statements](https://go.dev/tour/flowcontrol/6)?